aten/src/ATen/native/cuda/jit_utils.cpp

```diff
@@ -964,7 +966,11 @@ std::string generate_code(
 }
 
 //FIXME - this are defined in Loops.cuh, but including Loops.cuh here would lead to circular includes Loops.cuh -> CUDALoops.cuh -> jit_utils.h -> Loops.cuh
+#ifdef USE_ROCM
+#define THREAD_WORK_SIZE 8
+#else
 #define THREAD_WORK_SIZE 4
+#endif
```

Comment:
As per @jerrymannil, the reason for repeating this hardcoded value in this file is:
> one is used inside the jit code string template; other one is used outside of it. We are fixing it upstream in [[ROCm] Improvements for vectorized elementwise kernels by jerrymannil · Pull Request #143269 · pytorch/pytorch](https://github.com/pytorch/pytorch/pull/143269) 
where the  code string takes it as an argument